GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.cpp

```diff
@@ -0,0 +1,820 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "vulkanCore.h"
+#include "util/gvr_log.h"
+#include <assert.h>
+#include <cstring>
+
+VulkanCore* VulkanCore::theInstance = NULL;
+
+#define QUEUE_INDEX_MAX 99999
```

Comment:
Removed the magic number and use a test against queueFamilyCount + 1 to check if graphics queue is present.